### PR TITLE
Switch to deploy_cluster.yml

### PR DESCRIPTION
**Pull request by mtnbikenc**
As part of the playbook consolidation and refactoring, the new standard build playbook is playbooks/deploy_cluster.yml.

### CODE REVIEW

**Comment by michaelgugino** [comment]
LGTM

**Comment by sdodson** [comment]
These jobs are cross branch so we'll need to add a conditional check on the existence of the playbook.

**Comment by mtnbikenc** [comment]
Yep, short-term memory loss.  I'll update.

**Comment by mtnbikenc** [comment]
Updated for cross branch support.

File: continuous-upgrade/jobs/install-ocp.sh [hunk 1]
```diff
@@ -21,19 +21,24 @@ cd /data/src/github.com/openshift/aos-cd-jobs/
 git pull origin master
 /data/src/github.com/openshift/aos-cd-jobs/continuous-upgrade/actions/install_junit.sh
 sudo yum-config-manager --disable rhel-7-server-ose-3\*,li
+if [[ -s "${playbook_base}/deploy_cluster.yml" ]]; then
+    playbook="${playbook_base}deploy_cluster.yml"
```
Comment (sdodson) [inline continuous-upgrade/jobs/install-ocp.sh]: missing a / here?

File: continuous-upgrade/jobs/install-ocp.sh [hunk 2]
```diff
@@ -21,19 +21,24 @@ cd /data/src/github.com/openshift/aos-cd-jobs/
...
+else
+    playbook="${playbook_base}byo/config.yml"
```
Comment (sdodson) [inline continuous-upgrade/jobs/install-ocp.sh]: and here?

File: continuous-upgrade/jobs/install-ocp.sh (hunk 1 above)
Comment (sdodson) [reply continuous-upgrade/jobs/install-ocp.sh]: Mainly asking because the conditional has a / I don't know if playbook_base ends in a /

Comment (mtnbikenc) [reply continuous-upgrade/jobs/install-ocp.sh]: playbook_base does end in a '/'.  I'll fix these.

**Comment by mtnbikenc** [comment]
Corrected extra '/'.

**Comment by stevekuznetsov** [comment]
```
$ sjb/push-update-automatic.sh 
[INFO] Checking for existence of ami_build_origin_int_rhel_install...
[INFO] Checking for existence of merge_pull_request_jenkins_client_plugin...
[INFO] Checking for existence of merge_pull_request_jenkins_sync_plugin...
[INFO] Checking for existence of merge_pull_request_jenkins_openshift_login_plugin...
[INFO] Checking for existence of test_branch_image_registry_extended...
[INFO] Checking for existence of merge_pull_request_jenkins_plugin...
[INFO] Checking for existence of test_branch_jenkins_client_plugin...
[INFO] Checking for existence of test_branch_jenkins_sync_plugin...
[INFO] Checking for existence of test_branch_jenkins_openshift_login_plugin...
[INFO] Checking for existence of test_branch_openshift_ansible_logging_json_file...
[INFO] Checking for existence of test_branch_origin_extended_conformance_install_update...
[INFO] Checking for existence of test_branch_openshift_ansible_logging...
[INFO] Checking for existence of test_pull_request_jenkins_sync_plugin...
[INFO] Checking for existence of test_pull_request_jenkins_client_plugin...
[INFO] Checking for existence of test_branch_origin_extended_conformance_install_update_containerized...
[INFO] Checking for existence of test_pull_request_jenkins_openshift_login_plugin...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install...
[INFO] Checking for existence of test_branch_origin_extended_conformance_install_containerized...
[INFO] Checking for existence of test_branch_origin_extended_image_ecosystem...
[INFO] Checking for existence of test_branch_jenkins_plugin...
[INFO] Checking for existence of test_branch_origin_extended_conformance_install...
[INFO] Checking for existence of test_pull_request_origin_extended_image_registry...
[INFO] Checking for existence of test_pull_request_image_registry_extended...
[INFO] Checking for existence of test_pull_request_jenkins_plugin...
[INFO] Checking for existence of test_pull_request_origin_extended_conformance_install_update...
[INFO] Checking for existence of test_pull_request_origin_extended_conformance_install...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_with_status_check...
[INFO] Checking for existence of test_branch_origin_extended_image_registry...
[INFO] Checking for existence of test_pull_request_origin_extended_conformance_crio...
[INFO] Checking for existence of test_branch_origin_extended_conformance_crio...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_update_containerized...
[INFO] Checking for existence of test_branch_origin_extended_conformance_install_system_containers...
[INFO] Checking for existence of test_pull_request_openshift_ansible_logging...
[INFO] Checking for existence of test_pull_request_origin_aggregated_logging_ansible_json_file...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_update_system_containers...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_containerized...
[INFO] Checking for existence of test_branch_origin_extended_builds...
[INFO] Checking for existence of test_pull_request_origin_aggregated_logging_ansible...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_crio...
[INFO] Checking for existence of test_pull_request_origin_extended_builds...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_update...
[INFO] Checking for existence of test_branch_origin_extended_conformance_install_update_system_containers...
[INFO] Checking for existence of test_pull_request_origin_extended_image_ecosystem...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_system_containers...
[INFO] Updating test_pull_request_jenkins_plugin...
[INFO] Updating test_branch_origin_extended_conformance_install_system_containers...
[INFO] Updating test_branch_openshift_ansible_logging...
[INFO] Updating ami_build_origin_int_rhel_install...
[INFO] Updating test_pull_request_origin_aggregated_logging_ansible...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_containerized...
[INFO] Updating test_pull_request_jenkins_openshift_login_plugin...
[INFO] Updating test_branch_origin_extended_builds...
[INFO] Updating test_branch_origin_extended_conformance_install_containerized...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_update_system_containers...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_update_containerized...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_with_status_check...
[INFO] Updating test_pull_request_origin_extended_image_ecosystem...
[INFO] Updating test_branch_origin_extended_image_registry...
[INFO] Updating test_pull_request_origin_extended_conformance_install...
[INFO] Updating test_pull_request_origin_extended_conformance_crio...
[INFO] Updating merge_pull_request_jenkins_openshift_login_plugin...
[INFO] Updating test_pull_request_origin_extended_conformance_install_update...
[INFO] Updating test_branch_origin_extended_conformance_crio...
[INFO] Updating test_branch_origin_extended_conformance_install...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_update...
[INFO] Updating test_pull_request_origin_aggregated_logging_ansible_json_file...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_crio...
[INFO] Updating test_branch_image_registry_extended...
[INFO] Updating test_branch_jenkins_client_plugin...
[INFO] Updating test_pull_request_openshift_ansible_logging...
[INFO] Updating test_branch_openshift_ansible_logging_json_file...
[INFO] Updating test_branch_jenkins_sync_plugin...
[INFO] Updating merge_pull_request_jenkins_client_plugin...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install...
[INFO] Updating test_pull_request_origin_extended_builds...
[INFO] Updating test_branch_origin_extended_conformance_install_update_containerized...
[INFO] Updating test_branch_origin_extended_image_ecosystem...
[INFO] Updating test_pull_request_jenkins_client_plugin...
[INFO] Updating test_pull_request_origin_extended_image_registry...
[INFO] Updating merge_pull_request_jenkins_sync_plugin...
[INFO] Updating test_pull_request_jenkins_sync_plugin...
[INFO] Updating test_branch_origin_extended_conformance_install_update_system_containers...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_system_containers...
[INFO] Updating test_branch_origin_extended_conformance_install_update...
[INFO] Updating merge_pull_request_jenkins_plugin...
[INFO] Updating test_branch_jenkins_openshift_login_plugin...
[INFO] Updating test_branch_jenkins_plugin...
[INFO] Updating test_pull_request_image_registry_extended...
```

**Pull request merged**